<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>900</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Natureza da Operacao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Debito / Credito</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Notas de Entrada / Saida</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo de NF's</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(132, 168, 163);
color: rgb(33, 25, 255);
font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CFOP Cadastrados</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_inclusao">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Inclusao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_alteracao">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Alteracao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_exclusao">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exclusao</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>59</y>
      <width>901</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>INCLUSAO - ALTERACAO</string>
    </property>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>70</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>56</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Codigo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moperacao">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>70</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Tipo NF's</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Natureza da Operacao</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Entrada/Saida</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mdescr_op">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Deb./Cred.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_salvar">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>30</y>
       <width>90</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;file:///C:/Users/mebbc/OneDrive/Imagens/salvar.qrc.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> SALVAR</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>SALVAR.png</normaloff>SALVAR.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_cancelar">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> Cancelar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>SAIR.png</normaloff>SAIR.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>SAIR.png</normaloff>SAIR.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
